pjmedia/include/pjmedia/vid_stream.h

```diff
@@ -454,6 +454,18 @@ PJ_DECL(pj_status_t) pjmedia_vid_stream_send_rtcp_sdes(
 PJ_DECL(pj_status_t) pjmedia_vid_stream_send_rtcp_bye(
 						pjmedia_vid_stream *stream);
 
+
+/**
+ * Send RTCP PLI for the media stream.
```

Comment:
```suggestion
 * Send RTCP PLI for the video stream.
```

Reply:
2 API above this (send_rtcp_sdes() and _bye()) also uses media stream, probably because it's copied from audio stream, but the rest already more appropriately and specifically use the words "video stream".